### PR TITLE
Add Rocky Linux to installation list

### PR DESCRIPTION
**Pull request by brianclemens**

### CODE REVIEW

File: _includes/free-software-logos/rockylinux.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="4096" height="4096" viewBox="0 0 192 192" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (justdave) [inline _includes/free-software-logos/rockylinux.svg:1]: This actually renders on the website at 4096x4096, which is about 100 times the size of the rest of the logos.  It also displays in green while the rest of them are monochrome.  Is there a version of this logo that includes the name with it? The other logos in this section all include names with them.

**Review by justdave — CHANGES_REQUESTED** [review]

File: _includes/free-software-logos/rockylinux.svg (hunk 1 above)
Comment (brianclemens) [reply _includes/free-software-logos/rockylinux.svg:1]: lmao I didn't see that, fixing that now

Comment (brianclemens) [reply _includes/free-software-logos/rockylinux.svg]: And fixed. On a related note, [Boxy SVG](https://boxy-svg.com/) is wonderful.

**Comment by brianclemens** [comment]
Well this is embarrassing, I forgot we had switched back to Mantis. In my defense I'm not working on much sleep lol.

**Pull request closed without merging**